src/account/accountReducers.js

```diff
@@ -79,9 +79,8 @@ export default (state = initialState, action) => {
       ];
     }
     case LOGOUT: {
-      const newState = state.slice();
-      newState[0].apiKey = '';
-      return newState;
+      // Remove the active account
```

Comment:
It is not confusing, that is how a lot of multi-account apps work - Google, Slack etc. We have an 'X' for removal.